gemrb/GUIScripts/GUICommonWindows.py

```diff
@@ -266,7 +266,15 @@ def SetupMenuWindowControls (Window, Gears=None, CloseWindowCallback=None):
 	if Button:
 		Button.SetTooltip (OptionTip['Rest'])
 		Button.SetEvent (IE_GUI_BUTTON_ON_PRESS, RestPress)
-
+	elif rb == 11 and bg2:
+		#The 15 gap came from the guiw8.chu that is the network button
+		pos = ButtonOptionFrame["y"] + ButtonOptionFrame["h"] + 15
+		if pos > Window.GetFrame()["h"]:
```

Comment:
Is there ever not enough space?

Reply:
I don't know, just in case for by a little defensive. I can take it out.

Reply:
Since the smaller window packs do contain the button, they won't reach this code, so yeah, it can be removed. In the hypothethical case that it would be hit, buttons would overlap.